cmsmon-tools.spec

```diff
@@ -0,0 +1,76 @@
+### RPM cms cmsmon-tools 0.3.10
+## INITENV +PATH PYTHONPATH %i/${PYTHON_LIB_SITE_PACKAGES}
```

Comment:
there is nothing python related in this package , so why to set PYTHONPATH? Please remove

Reply:
done